tests/test_env.py

```diff
@@ -43,7 +43,7 @@ def test_get_args_and_env():
     assert result[1] == {
         "BM_SPIDER_JOB": "1-2-3",
         "BM_SPIDER_NAME": "demo",
-        "BM_API_HOST": "http://bitmaker-api.com",
+        "BM_API_HOST": "http://estela-api.com",
```

Comment:
The `BM` prefix of the env variables refers to `BitMaker`, we should use now `ESTELA` as the prefix. Please check all the files for the presence of env varriables. Take into account that this change implies a change in the API engine (function to start jobs).